src/modules/settings/components/settings/global/EmoteAutocomplete.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Panel from 'rsuite/Panel';
+import Toggle from 'rsuite/Toggle';
+import {registerComponent} from '../../Store.jsx';
+import {CategoryTypes, SettingIds} from '../../../../../constants.js';
+import styles from '../../../styles/header.module.css';
+import useStorageState from '../../../../../common/hooks/StorageState.jsx';
+
+function EmoteAutocomplete() {
+  const [value, setValue] = useStorageState(SettingIds.EMOTE_AUTOCOMPLETE);
+
+  return (
+    <Panel header="Emote Autocomplete">
+      <div className={styles.toggle}>
+        <p className={styles.description}>Typing : before text will attempt to autocomplete your emote</p>
+        <Toggle checked={value} onChange={(state) => setValue(state)} />
+      </div>
+    </Panel>
+  );
+}
+
+export default registerComponent(EmoteAutocomplete, {
```

Comment:
does this setting still make sense for twitch given it's part of the builtin?

Reply:
I believe the user should still be able to disable it

Reply:
why?

Reply:
should conflictions not be a conern?